### DISCUSSION TITLE
How much volume can AGH handle in a busy edu environment?

### DISCUSSION THREAD

**Discussion by dhcpserver** [AdguardTeam/AdGuardHome#3665] [General]
Been playing with an install on Debian and I'm impressed, it's really well done.

If I run It on some real hardware, any idea how many transactions/second it could handle?  I have a charter school I volunteer in and this would be fantastic to handle some on-prem special filtering before the traffic leaves the school.  I've been able to do some high volume with PiHole, but I really like this better.

Any thoughts?  Where would be the limitations, assuming rate-limiting was removed?

Thanks!

**Comment by ameshkov** [comment]
Tbh, it depends on the hardware. Generally, there's no specific limitation, it can be thousands of RPS.
